packages/server/src/seeder/utils.ts

```diff
@@ -147,6 +162,26 @@ export async function createArticlesPerUser(email: string, numberOfArticles: num
     // Create articles to the database
     // use the function generateFakeArticle to generate fake article
     // initially article should be created in draft form ( isPublished status set to false )
+
+    for (let i = 0; i < numberOfArticles; i++) {
+        const fakeArticle = await generateFakeArticle();
+        axios.post('/api/user/article', {
```

Comment:
Do not call the axios , use the Prisma client directly to add the data to database.

Reply:
Ok, that was my question, Just wanted to make sure that I use prisma directly

Reply:
Yes use prisma directly